dev-aws/kafka-shared/customer-billing/invoice-fulfillment/invoice-fulfillment.tf

```diff
@@ -0,0 +1,21 @@
+resource "kafka_topic" "invoice-fulfillment" {
+  name               = "bex.internal.bill_fulfilled"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    # keep data for 7 days
+    "retention.ms" = "604800000"
+    # allow max 1 MB for a message
+    "max.message.bytes" = "1048576"
+    "compression.type" = "zstd"
+    "cleanup.policy"   = "delete"
+  }
+}
+
+module "producer" {
```

Comment:
Do you need also a consumer? Or you'll add it later in another PR?

Reply:
For now the consuming service implementation has not started, so will add a consumer at a later date on a different PR.